core/controllers/profile.py

```diff
@@ -69,10 +69,12 @@ def get(self, username):
         if user_contributions:
             created_exp_summary_dicts = (
                 summary_services.get_displayable_exp_summary_dicts_matching_ids(
-                    user_contributions.created_exploration_ids))
+                    user_contributions.created_exploration_ids,
+                    user_services.UserActionsInfo()))
```

Comment:
Ditto -- passing in an empty object is not a good idea. It feels like it means something when in fact it's just a sentinel value for None.